Fix the dsbevx signature: Q is an array

potocpav opened the pull request:
As documented in http://www.netlib.org/lapack/explore-3.1.1-html/dsbevx.f.html#DSBEVX.1

IvanUkhov commented:
Thanks a lot! Could you please fix the scripts in `bin` instead? I’ll then regenerate the Rust code.

potocpav commented:
I fixed the scripts, regenerated the Rust code and added a comment to indicate how the code is generated.

I found out that `Q` is a scalar iff it is an integer. The only exceptions are auxilliary functions `sladiv` and `dladiv`, which are not exported by `lapack-sys` - they are irrelevant.
